src/content/1.7/basics/keeping_up-to-date/preliminary_task.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Preliminary task
```

Comment:
I think we can change this to:
- title: "How to backup PrestaShop"
- menuTitle: "Backup your shop"